packages/terra-tag/tests/nightwatch/IconAndTextTag.jsx

```diff
@@ -0,0 +1,15 @@
+import React from 'react';
+
+import IconBookmark from '../../../terra-icon/lib/icon/IconTile';
+import Tag from '../../lib/Tag';
+
+const icon = <IconBookmark />;
+const text = 'Long Long Long Long Long Long Long Long Long Long Long Long Long Long Long Long Long Long';
```

Comment:
You could use lorem ipsum text here.

Reply:
Updated it [here](https://github.com/cerner/terra-core/pull/1306/commits/177991d1fd0f601092750c29c037a85182dc3032).